node/runtime/pangoro/src/pallets/evm.rs

```diff
@@ -54,13 +56,15 @@ where
 	}
 
 	pub fn used_addresses() -> sp_std::vec::Vec<H160> {
-		sp_std::vec![1, 2, 3, 4, 9, 21, 26, 27].into_iter().map(|x| addr(x)).collect()
+		sp_std::vec![1, 2, 3, 4, 9, 21, 26, 27, 28, 29].into_iter().map(|x| addr(x)).collect()
```

Comment:
Lets start with 2048 from now.

Reply:
I think it is better to pin
BLS --> address(2046)
MPT --> address(2047)
Cause we will remove the two precompile contract after finish the solidity version bls and mpt lib.
Leave the 2048 empty to let more important precompile in.

Reply:
stable custom precompiles which is ready for users will start from 1024, 2048+ is for experimental precompiles.

Reply:
Make sense